R/dpinput_map.R

```diff
@@ -1,59 +1,58 @@
 #' @title Map data input into data product workflow
-#' @description This is helper function to help manage the context of 
+#' @description This is helper function to help manage the context of
 #' input_files folder
 #' @param project_path path to the project
 #' @return input_map a data.frame that maps the content of the input_files to be
 #'  evaluated
 #' @export
 dpinput_map <- function(project_path){
-  
-  input_dir <- glue::glue("{project_path}/input_files/") 
+
+  input_dir <- glue::glue("{project_path}/input_files/")
   # TODO: extend the same workflow seamlessly to a single data file
   if(!fs::dir_exists(input_dir))
     stop(cli::format_error(glue::glue("input_dir {input_dir} specified does ",
                                       "not exist. Check the path")))
-  
-  if(length(fs::dir_ls(input_dir))==0)
-    stop(cli::format_error("Input_dir is empty"))
-  
-  
-  
+
+  if(length(fs::dir_ls(input_dir))==0){
+    message("input_files directory is empty. It is OK to have an empty input_files directory if there is no need to update input data.")
+  }
```

Comment:
For consistency of the look and feel, it may be good to use `cli` package for formatting all console outputs (i.e. messages, errors, warnings etc.)